cmd/status/printers/printers.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package printers
+
+import (
+	"fmt"
+	"io"
+
+	"sigs.k8s.io/cli-utils/cmd/status/print"
+	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/collector"
+)
+
+func CreatePrinter(printerType string, collector *collector.ResourceStatusCollector, w io.Writer) (print.Printer, error) {
```

Comment:
i'll only mention it here, but feel free to keep it at 80 chars.
https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#Line_Length

Break wherever you like, as long as the Go formater respects it.

Reply:
I have updated the PR to stay within 80 chars.